projects/igniteui-angular/src/lib/grids/grid/grid-filtering.spec.ts

```diff
@@ -649,6 +655,18 @@ describe('IgxGrid - Filtering actions #grid', () => {
         expect(grid.rowList.length).toEqual(1);
     }));
 
+    it('should exclude null and undefined values when filter by \'false\' and column dataType is not set', fakeAsync(() => {
+        fix = TestBed.createComponent(IgxGridESFComponent);
+        fix.detectChanges();
+        grid = fix.componentInstance.grid;
+
+        grid.filter('Released', false, IgxStringFilteringOperand.instance().condition('equals'), true);
+        fix.detectChanges();
+        expect(grid.rowList.length).toEqual(2);
+        expect(!(grid.getCellByColumn(0, 'Released').value));
+        expect(!(grid.getCellByColumn(1, 'Released').value));
```

Comment:
It seems that the expected values are missing.
```suggestion
        expect(grid.getCellByColumn(0, 'Released').value).toBe(false);
        expect(grid.getCellByColumn(1, 'Released').value).toBe(false);
```